lib/FileProcessor.js

```
@@ -113,8 +113,10 @@ class FileProcessor {
         }
       }
       this.data.push(resultObject);
-      //TODO: errors aren't bubbling up
-      //need to think about this since we don't want to throw a parse error.  or do we?
+      // display any parse errors but do not bubble them up:
+      if (err) {
+        console.log(`${err.name} in file ${file}: ${err.message}`);
```

we should bubble the error back up

---

make sure to add a test too (if none)